poetry.lock

```diff
@@ -1,13 +1,11 @@
-# This file is automatically @generated by Poetry 2.0.1 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.8.5 and should not be changed by hand.
```

Comment:
looks like this was generated with the wrong poetry version, we just fixed this in #1656
do you mind rebasing and running `poetry lock`?